java/com/google/copybara/git/GitModule.java

```
@@ -341,6 +342,16 @@ public GitOrigin origin(
     checkSubmoduleConfig(submodules, excludedSubmoduleList);
     String fixedUrl = fixHttp(url, thread.getCallerLocation());
     CredentialFileHandler credentialHandler = getCredentialHandler(fixedUrl, credentials);
+
+    boolean isGitHubUrl = false;
+    try{
+        GitHubOptions githubOptions = options.get(GitHubOptions.class);
+        GitHubHost ghHost = githubOptions.getGitHubHost(url);
+        isGitHubUrl = ghHost.isGitHubUrl(url);
+    } catch (EvalException e) {
```

empty catch blocks are a bit of an anti-pattern, and I'd like to avoid having these if possible. is there any way you can refactor this into a method in GitHubOptions that returns whether a url belongs to one of the GH hosts?

if you must keep the empty catch block, please leave a comment explaining why we catch and swallow the exception